packages/mobile/src/app/actions.ts

```diff
@@ -33,6 +33,7 @@ export enum Actions {
   APP_MOUNTED = 'APP/APP_MOUNTED',
   APP_UNMOUNTED = 'APP/APP_UNMOUNTED',
   VERIFICATION_MIGRATION_RAN = 'APP/VERIFICATION_MIGRATION_RAN',
+  SET_GOOGLE_PLAY_SERVICES_AVAILABILITY = 'APP/SET_GOOGLE_PLAY_SERVICES_AVAILABILITY',
```

Comment:
Though the current code base is not a great example of this, moving forward we want to model actions as events instead of setters. See why https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters

```suggestion
  GOOGLE_PLAY_SERVICES_AVAILABILITY_CHECKED = 'APP/GOOGLE_PLAY_SERVICES_AVAILABILITY_CHECKED',
```

Reply:
Thanks, fixed in 0128da9886fedc94b4bd4204011f986c779130af